lib/cgo/cipher.crypto.hw.go

```diff
@@ -0,0 +1,150 @@
+package main
```

Comment:
Put these back to their original location . This makes it impossible to auto-generate code out of skycoin/skycoin function signatures. Find another way to get this done .